ppx/ftp.py

```diff
@@ -205,15 +217,21 @@ def _parse_files(self):
 
         return files + new_files, dirs + new_dirs
 
-    def download(self, files, dest_dir, force_=False, silent=False):
+    def download(
+        self,
+        files: Union[str, List[str]],
+        dest_dir: Union[Path, CloudPath],
+        force_=False,
+        silent=False,
+    ):
         """Download the files
 
         Parameters
         ----------
-        remote_file : str
-            The file to download.
-        out_file : pathlib.Path object
-            The local file.
+        files : [str]
```

Comment:
I've been documenting in the NumPy style, so let's change this to `files : list of str`